pkg/webhooks/webhooks.go

```diff
@@ -66,6 +69,9 @@ func Initialize(injectCloudProvider func(cloudprovider.Context) cloudprovider.Cl
 	ctx := injection.LoggingContextOrDie(component, config, cmw)
 	ctx = knativeinjection.WithNamespaceScope(ctx, system.Namespace())
 
+	// TODO: Consider customizing these observability metrics later by creating an observability ConfigMap
```

Comment:
Can you add a note that knative currently crashes if you don't include this, so someone doesn't remove it inadvertantly?